custom_libs/subliminal_patch/providers/jimaku.py

```diff
@@ -0,0 +1,302 @@
+from __future__ import absolute_import
+
+import io
+import logging
+import re
+import time
+import traceback
+import urllib.parse
+import rarfile
+import zipfile
+import requests
+
+from requests import Session
+from subliminal import __short_version__
+from subliminal.exceptions import ConfigurationError, AuthenticationError
+from subliminal.utils import sanitize
+from subliminal.video import Episode, Movie
+from subliminal_patch.providers import Provider
+from subliminal_patch.subtitle import Subtitle
+from subliminal_patch.exceptions import APIThrottled
+from subliminal_patch.providers.utils import get_subtitle_from_archive
+from guessit import guessit
+from functools import cache
+from subzero.language import Language
+
+logger = logging.getLogger(__name__)
+
+class JimakuSubtitle(Subtitle):
+    '''Jimaku Subtitle.'''
+    provider_name = 'jimaku'
+    
+    hash_verifiable = False
+
+    def __init__(self, video, subtitle_id, subtitle_url, subtitle_filename):
+        # Override param 'language' as it could only ever be "ja"
+        super(JimakuSubtitle, self).__init__(Language("jpn"))
```

Comment:
You should not hardcode language but instead detect them properly since you can have multiple languages subtitles.

Reply:
No longer hardcoded, instead the language now gets determined through `_try_determine_subtitle_languages()`.